src/core/Tree/index.ts

```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+export { Tree } from './Tree';
+export type { TreeProps } from './Tree';
+export default './Tree';
```

Comment:
Not sure if default export is needed here.. Something useful it gives in this case?

Reply:
Not sure, its what is added by default when running the `CreateComponent` script

Reply:
Removed default